tests/keras/backend/backend_test.py

```diff
@@ -506,9 +506,16 @@ def test_elementwise_operations(self):
         check_single_tensor_operation('std', (4, 2), WITH_NP, axis=1, keepdims=True)
         check_single_tensor_operation('std', (4, 2, 3), WITH_NP, axis=[1, -1])
 
-        check_single_tensor_operation('logsumexp', (4, 2), WITH_NP)
-        check_single_tensor_operation('logsumexp', (4, 2),
-                                      WITH_NP, axis=1, keepdims=True)
+        check_single_tensor_operation('logsumexp', (3,), WITH_NP, axis=0)
+        for shape in [(1, 3), (2, 1), (4, 2)]:
```

Comment:
In order to avoid this `for` loop, I think it would be better to have the test for `logsumexp` be a separate function, with a parameterization decorator over axes and shapes.